packages/playwright-core/src/server/isomorphic/cssTokenizer.ts

```diff
@@ -131,7 +131,8 @@ export function tokenize(str1: string): CSSTokenInterface[] {
   const donothing = function() { };
   const parseerror = function() {
     // eslint-disable-next-line no-console
-    console.log('Parse error at index ' + i + ', processing codepoint 0x' + code.toString(16) + '.'); return true;
+    // Language bindings don't like writing to stdout!
+    // console.log('Parse error at index ' + i + ', processing codepoint 0x' + code.toString(16) + '.'); return true;
```

Comment:
Can we still print those when under test?

Reply:
would be a lot of plumbing of `isUnderTest` and introduce a `logErrors: boolean` based on `isUnderTest` since `cssTokenizer` is isomorphic.

As of today we'd throw the following: `"Unsupported token \\"BADSTRING\\" while parsing selector \\"div:has-text(\\"A\\nA\\")\\"`

Reply:
Let's ask @dgozman if he is comfortable dropping these or we need to wire them...

Reply:
Seems safe, because we always return something like `BadStringToken()` and throw later on.